openmc/mesh.py

```diff
@@ -961,6 +961,44 @@ def build_cells(self, bc=None):
 
         return root_cell, cells
 
+
+    def get_index_where(self, value: float, basis: str = 'xy'):
+        """Gets the mesh cell index nearest to the specified axis value.
```

Comment:
```suggestion
        """Gets the mesh cell index nearest to the specified axis value for the basis provided.
```
Perhaps this docstring helps explain better